timechart/displays/main_display.py

```diff
@@ -430,6 +430,14 @@ def __init__(self, parent=None, args=[], macros=None, show_pv_add_panel=True, co
         self.time_span_limit_seconds = None
         self.data_sampling_mode = ASYNC_DATA_SAMPLING
 
+        parser = argparse.ArgumentParser(description='TimeChart')
+        parser.add_argument("--pvs", help="Launch TimeChart with PVs loaded from command line", nargs='*')
+        args = parser.parse_args()
+        self.pv_arg_list = args.pvs
+        if len(self.pv_arg_list):
```

Comment:
Another tip - `if []:` is effectively `if False:` and `if [x]:` is effectively `if True`.
That is to say, you don't need to check `len(x)` here, you can just do:
```suggestion
        if self.pv_arg_list:
```